### DISCUSSION TITLE
How to fix button in TextInput on Safari

### DISCUSSION THREAD

**Discussion by goodlythink** [mantinedev/mantine#2042] [Q&A]
**mycode**
![Screen Shot 2565-08-08 at 12 17 16](https://user-images.githubusercontent.com/12431734/183346986-68998638-ffe1-44ce-ba49-2acc947e9fe4.png)

**when display on **Chrome not problem** but display on **Safari it's problem****
![Screen Shot 2565-08-08 at 12 38 08](https://user-images.githubusercontent.com/12431734/183347113-437948e9-bf2a-49af-bb20-4fb25d1c9d25.png)
![Screen Shot 2565-08-08 at 12 38 46](https://user-images.githubusercontent.com/12431734/183347123-e46fbad3-ca01-4bc0-9217-73940bdcb3ed.png)

**Comment by rtivital** [comment]
Try increasing `rightSectionWidth`, if that does not help set `<Group noWrap />`

**Reply by goodlythink** [reply]
**noWrap** is work.
but google chrome space behind more than safari

![Screen Shot 2565-08-09 at 10 04 07](https://user-images.githubusercontent.com/12431734/183554759-95d563df-e4bd-4254-80f3-cf5baf3703ba.png)


![Screen Shot 2565-08-09 at 10 02 14](https://user-images.githubusercontent.com/12431734/183554775-07d414d3-a947-4b7a-ae63-dee3bf6e95af.png)

**Reply by rtivital** [reply]
It looks the same on Chrome and Safari on codesandbox – https://codesandbox.io/s/elastic-sutherland-j6sjw7?file=/src/App.tsx

**Reply by goodlythink** [reply]
thank you it's work
